index.html

```diff
@@ -36,14 +36,15 @@
     <div class="cover">
       <header class="header">
         <h1 class="logo"><a href="#">My Logo</a></h1>
-        <ul class="nav-bar-links">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+        <ul id="navbtn" class="nav-bar-links">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
         </ul>
-        <nav class="humberger-menu">
+        <nav id="hum" class="humberger-menu">
           <a href="#"><i class="fa fa-bars fa-lg"></i></a>
         </nav>
+        <span class="toggle-nav" id="openbtn">&#9776;</span>
```

Comment:
- [ ] By the look of things, your mobile menu is not set up properly as it should be based on your chosen design. Firstly, your hamburger icon is at the centre of the page. I would appreciate you working on it.

Your Design | Template Design |
:---------------:|:----------------------:|
![a1](https://user-images.githubusercontent.com/39523312/156269672-59fef84f-304d-4731-bd11-84c854ae7192.png)|![a3](https://user-images.githubusercontent.com/39523312/156269745-d1de30df-6a15-40db-bfde-20db9a10bad6.png)


Secondly, when a user clicks on the hamburger menu icon, an overlay should be displayed with the nav links. Please work on this as well.

Your design | Template Design |
:---------------:|:----------------------:|
![a4](https://user-images.githubusercontent.com/39523312/156269997-96f8d7ee-d313-4c0a-86f3-841f0c49af71.png)|![a 2](https://user-images.githubusercontent.com/39523312/156269682-57303de2-12e8-45e7-a23b-cbf358db12c1.png)